mopidy/core/library.py

```diff
@@ -313,8 +332,14 @@ def search(self, query=None, uris=None, exact=False, **kwargs):
 
         results = []
         for backend, future in futures.items():
-            try:
-                results.append(future.get())
+            try:  # TODO: fix all these cases so we can use common helper
+                result = future.get()
+                if result is not None:
```

Comment:
Because of this `is None` check, the return can be changed to simply `return results`, I think.